### ISSUE TITLE
[DOC] toNumber is not a method

### ISSUE THREAD

**Issue by joepetrowski** [polkadot-js/api#1581]
The example here uses `toNumber()` as a method but it does not exist: https://polkadot.js.org/api/start/api.consts.html

![image](https://user-images.githubusercontent.com/25483142/69047691-0fd13980-09fc-11ea-8f52-e74c2954b63c.png)

![image](https://user-images.githubusercontent.com/25483142/69047644-f62ff200-09fb-11ea-8cdf-37191c05c552.png)

**Comment by jacogr** [comment]
![image](https://user-images.githubusercontent.com/1424473/69048025-c7fee200-09fc-11ea-868e-c07add79414e.png)

**Comment by joepetrowski** [comment]
Must be my VS Code environment... sorry about that. New to playing with this.

**Comment by jacogr** [comment]
So it does exist. The issue with all these is that they return the base codec interface, not quite the specific. It is a WIP to get the typings right. (It is supposed to be fixed, but does not appear yet). 

TL;DR Yes, it is actually on the object. No, the typings are not correct as exposed.

cc @amaurymartiny

**Comment by amaurymartiny** [comment]
Will be solved by https://github.com/polkadot-js/api/issues/1490
